src/Type/Accessory/AccessoryArrayListType.php

```diff
@@ -139,7 +139,7 @@ public function unsetOffset(Type $offsetType): Type
 
 	public function flipArray(): Type
 	{
-		return $this;
+		return new MixedType();
```

Comment:
shouldn't this be more precise, like `ArrayType($valueType, $positiveInt)`?

Reply:
I think that shouldn't be necessary because this type should come intersected with an array and intersecting with mixed then just removes it basically. the simpler the better, I think, but I'm also not sure and @rvanvelzen just recently improved performance by not being too precise in type specification. maybe he can explain it better :)

Reply:
if it passes a non-constant test like

```
/**
 * @param list<string> $array
 */
function foo7($array)
{
	$flip = array_flip($array);
	assertType('array<string int<0, max>>', $flip);
}
```

we are good to go :)